### ISSUE TITLE
New Machbook IOS

### ISSUE THREAD

**Issue by TawfikDaim** [TawfikDaim/home_automation#31]

**Comment by TawfikDaim** [comment]
1. Install UTM, setup a new VM to be host network and using home assistant vdo iso file
2. login to homeassistant e.g. http://192.168.1.99:8123/ and assign a static IP address from Network setting in home assistant Activity
4. Ngnx and duckdns   https://youtu.be/bADNTSF4Fh4
      
      port forwarding
      First router will only allow port 443 to the IP of second router (192.168.1.11)
      Second router will allow both 433 and 8123 (since this router is already local) to the IP of VM of homeassistant 10.195.1.98
      
      tested when from outside and inside
      
      http:
      use_x_forwarded_for: true
      trusted_proxies:
      - 172.30.33.0/24
      ip_ban_enabled: true
      login_attempts_threshold: 5
      IP ban is not working as it automatically blocks local IPs ??
5. enable advanced

**Comment by TawfikDaim** [comment]
2. login to homeassistant e.g. http://192.168.1.99:8123/ and assign a static IP address from Network setting in home assistant Activity

**Comment by TawfikDaim** [comment]
5. enable advanced

**Comment by TawfikDaim** [comment]
install VSC, Samba share, SSH

**Comment by TawfikDaim** [comment]
FLO integration 

Water Shutoff devioce frpm Flo by Moen

Just add integration and sign in using FLO credentials (you need to buy the device and set it up using APK on an android device as its not supporting IOS)

**Comment by TawfikDaim** [comment]
**Copy any dahsboard to the new instance**
It’s a bit more work than a simple button click, but you can copy/paste an entire dashboard using the Edit Dashboard > Raw Configuration editor in the menu.

Issues: åå
1) if theme was not available it will not work and needs to be edited to add an available theme in new home assistant
2) some sonoff devices will come as Switch or Light and if it was used as the different type it must be corrected to work probeperly (good news is that only the identifier switch or light needs to change but the rest of the id remains the same)

**Comment by TawfikDaim** [comment]
Presence and location detection
Use iCloud integration for iPhone
assign the person to the device_tracker entity of the iCloud integration

**Comment by TawfikDaim** [comment]
**Ongoing** 

Use person presence in house as a condition for running automations, 
e.g scenarios

1. When no one is at home trigger an alarm if any motion sensor inside house detected motion (inside rooms, office)
2. When no one is at home trigger an alarm if camera sensor inside office detected motion 
3. Greeting when someone has arrived -> send message to Alexa 
4.

**Comment by TawfikDaim** [comment]
Next _ Alexa integration 

Alexa, who is home ?
Alexa, add item to shopping list

**Comment by TawfikDaim** [comment]
Automation for detecting movement when a person is away, this can be extended to be a mechanzim to lock home (close all lights excpet Fridge/automation/wifi), close all heating. close water valve at 11PM nightly while no one is at home), 
also can be used the other way around to open heating in winter when there is anyone home.

alias: Motioneye detected motion in office
description: ""
trigger:
  - platform: webhook
    webhook_id: Office169931
    allowed_methods:
      - POST
      - PUT
    local_only: false
condition:
  - condition: time
    after: "22:30:00"
    before: "07:00:00"
    enabled: false
action:
  - service: notify.mobile_app_tawfiks_iphone
    data:
      message: Office camera detected a motion
      title: Motioneye
    enabled: false
  - type: turn_on
    device_id: 2296691929c0872f42acb2deca95a4ce
    entity_id: 7bbeca76bc7ce34fa736118f03dc7831
    domain: light
    enabled: false
  - delay:
      hours: 0
      minutes: 0
      seconds: 50
      milliseconds: 0
    enabled: false
  - type: turn_off
    device_id: 2296691929c0872f42acb2deca95a4ce
    entity_id: 7bbeca76bc7ce34fa736118f03dc7831
    domain: light
    enabled: false
  - service: camera.snapshot
    target:
      device_id:
        - dcb549ec660ffc6632dc37ddf1cca46e
    data:
      filename: /config/www/tmp/snapshot-office.jpg
    enabled: false
  - service: notify.mobile_app_tawfiks_iphone
    data:
      data:
        image: http://192.168.1.25:8123/local/tmp/snapshot-office.jpg
      message: Movement in office
    enabled: false
  - if:
      - condition: state
        entity_id: person.tawfik_daim
        state: not_home
    then:
      - service: camera.snapshot
        target:
          device_id:
            - dcb549ec660ffc6632dc37ddf1cca46e
        data:
          filename: /config/www/tmp/snapshot-office.jpg
      - service: notify.mobile_app_tawfiks_iphone
        data:
          data:
            image: http://192.168.1.25:8123/local/tmp/snapshot-office.jpg
          message: Movement in office
mode: single

**Comment by TawfikDaim** [comment]
Install iCloud3 for tracking iPhone devices

**Manual Installation from the iCloud3 Repository Releases Page**

1. Download the icloud3.zip (or icloud3_v3.zip) file from the gcobb321/icloud3_v3 iCloud3 GitHub Repository Releases page [here](https://github.com/gcobb321/icloud3_v3/releases). Selects Assets at the bottom, then the zip file. The file save screen is displayed, select the location on your computer and save the zip file.
2. Unzip the file into the config/custom_components/icloud3 directory on your Home Assistant server (ex.: Raspberry Pi)
3. Restart Home Assistant

**Comment by TawfikDaim** [comment]
Theme

https://github.com/tgcowell/waves

**Comment by TawfikDaim** [comment]
Tuya Smart Lock

https://www.home-assistant.io/integrations/tuya

![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/8ee88f76-9632-4e4e-ab0b-c92d4a7a1224)

**Comment by TawfikDaim** [comment]
Mushroom Cards

**Comment by TawfikDaim** [comment]
TT lock
[
](https://github.com/jbergler/hass-ttlock)

![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/38d93f6f-9d04-4a49-b746-bb65956ed9b7)

**Comment by TawfikDaim** [comment]
Shopping List , Amazon Todoist   (ON HOLD)


1) install pyscript from Under HACS -> Integrations, select "+", search for pyscript and install it.
2) (https://github.com/custom-components/pyscript/tree/master)
3) Todoist API key ([Integrations (todoist.com)](https://todoist.com/prefs/integrations))  82977fb625984d1c53a10ee78104ca815d39f180
4) Todoist project ID (when on your Alexa Shopping List project, use the ID at the end of the URL)

**Comment by TawfikDaim** [comment]
hue 

https://www.youtube.com/watch?v=fgeZ1O6J4jc

# Amazon Echo example configuration.yaml entry
emulated_hue:
  type: alexa
  host_ip: 192.168.1.13
  listen_port: 80
  expose_by_default: false
  entities:
    switch.sonoff_100157b239_1:
      name: 'Office'
      hidden: false
    switch.sonoff_100157b239_2:
      name: 'Tawfik'
      hidden: false

**Comment by TawfikDaim** [comment]
Dashborad and theme 

add one section to show each of:
- https://www.google.com/search?q=add+background+image+in+dashboard+in+ho%2Ce+assistant&rlz=1C5CHFA_enEG1080EG1091&oq=add+background+image+in+dashboard+in+ho%2Ce+assistant&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIJCAEQIRgKGKAB0gEJMTI0NDFqMGo0qAIAsAIA&sourceid=chrome&ie=UTF-8#fpstate=ive&vld=cid:b4a8fa6e,vid:RcHoO2nYsVw,st:0
- https://youtu.be/6hemFbrJuk8?si=wb7UD5REwTGqR1M5
scroll vieiw and tabbed view https://youtu.be/9cm2CBBdr_I?si=JySr6WSOCMfSgN9U
- https://www.youtube.com/watch?v=RcHoO2nYsVw&t=497s

**Comment by TawfikDaim** [comment]
Alexa integration is not working as I changed duckDNS on 192,168,1,25 now need to reinstall again on new HA

**Comment by TawfikDaim** [comment]
try https://www.home-assistant.io/integrations/huawei_lte/
presence detection from router

**Comment by TawfikDaim** [comment]
**MotionEye** 
- Setup motion eye, configurig each camer settings below
- Camera integration, then add each camera by RTSP 



![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/71805094-5442-400f-af9e-03d1557d7ea3)



![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/9acae00a-1efa-402f-b1a4-e48a79def605)




![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/39169739-3366-4195-a3b3-df6c56e456f7)





![Image](https://github.com/TawfikDaim/home_automation/assets/32592408/7b4d0b18-f687-4be3-a720-664b52423617)
